cypress/e2e/landing-spec.cy.js

```diff
@@ -35,23 +35,24 @@ it('has an overview blurb', () => {
 })
 
 it('has featured local businesses', () => {
-  
+  cy.get('[data-cy="featured-cards"]')
+  .within(() => {
+    cy.get('[data-cy="business-card"]').first()
+    .should('contain', 'The Buff Restaurant').and('contain', 4.5)
+    .find('img').should('have.attr', 'src').should('include', 'https://s3-media2.fl.yelpcdn.com/bphoto/p95cYM_TgNOlHvw_cf0SAw/o.jpg')
```

Comment:
Make sure to indicate on your README that your work is tested for accessibility as well.